android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/MapSession.kt

```diff
@@ -42,6 +48,19 @@ class MapSession : Session() {
       mapScreen.setMapCameraController(carCameraController)
       loadStyle(surface)
       initLocationComponent(surface)
+
+      surface.addWidget(logo)
+      surface.addWidget(compass)
+
+      surface.getMapboxMap().apply {
+        addOnCameraChangeListener {
```

Comment:
guess we need to remove it somewhere as well

Reply:
This is the example implementation and it will be reworked after #1004 will be merged, I'd kept as is (the lifecycle logic will be massively reworked there).